homeassistant/components/supla/switch.py

```diff
@@ -0,0 +1,40 @@
+"""Support for Supla cover - curtains, rollershutters etc."""
+import logging
+from pprint import pformat
+
+from homeassistant.components.switch import SwitchDevice
+from homeassistant.components.supla import SuplaChannel
+
+DEPENDENCIES = ["supla"]
```

Comment:
Please remove this.

Reply:
I removed it from the code for other device type as well, is it alright?